app/controllers/people_controller.rb

```diff
@@ -73,4 +72,8 @@ def set_person
   def person_params
     params.require(:person).permit(:name, :user_id)
   end
+
+  def create_person
+    @person = Person.new(person_params)
+  end
```

Comment:
Created before action for "create" method to prevent Metrics/AbcSize rubocop fail